doc/source/howtos/upgrade_guide.rst

```diff
@@ -4,14 +4,16 @@
 Upgrading Percona XtraDB Cluster
 ================================
 
-This guide describes the procedure for upgrading |PXC| version 5.6
-to the latest version 5.7 without downtime (*rolling upgrade*).
-This means that replication between nodes will continue during the upgrade.
-
-Rolling upgrade from earlier versions to 5.7 is not supported.
-If you are running |PXC| version 5.5, it is recommended to shut down all nodes,
-then remove and re-create the cluster from scratch.
-Alternatively, you can perform a `rolling upgrade from PXC 5.5 to 5.6 <https://www.percona.com/doc/percona-xtradb-cluster/5.6/upgrading_guide_55_56.html>`_,
+This guide describes the procedure for upgrading |PXC| without downtime
+(*rolling upgrade*) to the latest 5.7 version. Rolling upgrade means no
+need to take down complete cluster during the upgrade.
+
+Both major upgrade (from 5.6 to 5.7 version) and minor one (from 5.7.x
+to 5.7.y)  can be done in this way. Rolling upgrade to 5.7 from versions
+older than 5.6 is not supported. Therefore if you are running |PXC| version
```

Comment:
s/is not/are not/

Reply:
"Both major upgrade (from 5.6 to 5.7 version) and minor one (from 5.7.x
+to 5.7.y)"

Isn't this a bit complicated procedure to follow for minor version upgrade, yum/apt upgrade should work instead of manually stopping/upgrading each node.

Reply:
@hrvojem 

If I understand this correctly then only following 2 commands can be replaced
$ sudo apt-get remove percona-xtrabackup* percona-xtradb-cluster*
$ sudo apt-get install percona-xtradb-cluster-57

with
sudo apt-get upgrade percona-xtradb-cluster-57

(Given your expertise in this field we can simply go ahead with your suggestion).

But need to stop the server and need to run mysql-upgrade in standalone is needed. Right?

Reply:
@kbauskar 

IMO, to avoid the confusion we should split the minor from major upgrade docs, either in separate page or just a separate section. 

> But need to stop the server and need to run mysql-upgrade in standalone is needed. Right?

We can advise on stopping the service (but upgrade process should take care of it automatically), yes running mysql_upgrade should be mentioned.